taxonomy/signals/handlers.py

```diff
@@ -85,3 +88,20 @@ def handle_xblock_published(**kwargs):  # pylint: disable=unused-argument
         LOGGER.error('[TAXONOMY] Received null or incorrect data from XBLOCK_PUBLISHED.')
         return
     update_xblock_skills.delay([xblock_data.usage_key])
+
+
+@receiver(XBLOCK_SKILL_VERIFIED)
+def handle_xblock_verified(**kwargs):
+    """
+    Handle signal and trigger task to update xblock skills verified and ignored count.
+    """
+    LOGGER.info('[TAXONOMY] XBLOCK_SKILL_VERIFIED signal received')
+    xblock_data = kwargs.get('xblock_info', None)
+    if not xblock_data or not isinstance(xblock_data, XBlockSkillVerificationData):
+        LOGGER.error('[TAXONOMY] Received null or incorrect data from XBLOCK_SKILL_VERIFIED.')
+        return
```

Comment:
Does it make sense to add a check to see if verified_skills or ignored_skills is empty?